Week-2/Assignment_2_1/styles/style.css

```diff
@@ -0,0 +1,47 @@
+* {
+  box-sizing: border-box;
+}
+
+header {
+  margin-top: 5%;
+  text-align: center;
+}
+
+h1,
+h2 {
+  font-weight: 400;
+  font-family: DauphinPlain;
+}
+h3,
+h4,
+h5 {
+  font-weight: 400;
+}
+
+#section {
+  padding: 40px 40px 20px 20px;
+  float: left;
+  width: 50%;
+}
+
+#aside {
+  padding: 40px 40px 20px 20px;
+  float: left;
+  width: 50%;
+}
+
+img {
+  margin-top: 5%;
+  margin-left: 25%;
+  margin-right: auto;
+}
+
+form {
+  margin-left: 20px;
+}
+@media screen and (max-width: 800px) {
```

Comment:
This is one way of doing it.. can we make use of flex to achieve the same as part of learning? instead of float left and media queries try to make use of flex concepts to achieve the same

Reply:
Flex applied
![image](https://user-images.githubusercontent.com/108617696/182028126-62ecb331-0791-4bf8-a05f-429ba530e0f2.png)